ui/appui-react/src/appui-react/tools/CoreToolDefinitions.tsx

```diff
@@ -38,6 +38,7 @@ import cameraAnimationDisabledIcon from "@itwin/itwinui-icons/icons/camera-anima
 // istanbul ignore next
 export class CoreTools {
   /** Get the CustomItemDef for PopupButton
+   * @deprecated
```

Comment:
@NancyMcCallB  mentionned yesterday that this was available because Studio needed it, I missed part of the conversation but are we removing the keyin browser ?

Reply:
This is the original keyin browser, I think. The one that looks like the MicroStation browser.
The keyin palette is still available. We do need to coordinate with the Studio team, because having two difference browsers will cause confusion.